packages/block-editor/src/store/effects.js

```diff
@@ -219,6 +271,30 @@ export default {
 			)
 		);
 	},
+	MOVE_BLOCKS_DOWN: ( action, { getState } ) => {
+		const state = getState();
+		const settings = getSettings( state );
+
+		let direction = 'down';
+
+		if ( action.orientation === 'horizontal' ) {
+			direction = settings.isRTL ? 'left' : 'right';
```

Comment:
Is this correct? Wouldn't the actual move actions still be accurate given left moves to the left regardless of RTL?

Reply:
It is correct. "down" moves the block to the "following" position, whether it's vertical or horizontal. In LTR, the "following" horizontal position is right, while in RTL is left.